consumer-services/src/test/scala/za.co.absa.spline.consumer.service.internal/AttributeDependencySolverSpec.scala

```diff
@@ -0,0 +1,511 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.spline.consumer.service.internal
+
+import java.util.UUID
+
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import collection.JavaConverters._
+import za.co.absa.spline.consumer.service.internal.AttributeDependencySolver.resolveDependencies
+import za.co.absa.spline.consumer.service.internal.model.OperationWithSchema
+
+class AttributeDependencySolverSpec extends AnyFlatSpec with Matchers{
+
+  private val A, B, C, D, E, F, G = UUID.randomUUID()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'G' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/ABSA_OSS/spline/pullRequest?prid=4773990)